translations/base-fr.yaml

```diff
@@ -11,6 +11,16 @@ steamPage:
         Et en plus, vous devrez aussi produire de plus en plus pour satisfaire la demande. La seule solution est de construire en plus grand ! Au début vous ne ferez que découper les formes, mais plus tard vous devrez les peindre — et pour ça vous devrez extraire et mélanger des couleurs !
 
         En achetant le jeu sur Steam, vous aurez accès à la version complète, mais vous pouvez aussi jouer à une démo sur shapez.io et vous décider ensuite !
+ <<<<<<< patch-2
```

Comment:
This PR is not merged properly

Reply:
sorry if i am annoying but i dont understand what is wrong...

Reply:
So what basically happened is that the lines you modified were modified in the master branch after your PR. This means they can't be merged automatically. These are called conflicts. Now it's your turn to make a manual merge.
GitHub has added some syntax to mark the conflicts for you.
<<<<<<<<    means that the conflict starts here
patch-2   marks that the following is your version of the file
=======  divides your version and the master of tobspr
>>>>>>>>> master    marks the end of the conflict
So what you have to do in order to merge it is:
- Look at both versions of the file as marked
- Decide which version is correct or better
- Delete the other version so only one is left. It doesn't matter which one or if you mixed them
- Delete all of the marks GitHub has added
I'll give you an example:

This is what it looks like now:
```
<<<<<<< patch-2
     key1: This is the correct translation
=======
   key2: This is a wrong translation and it is not indented correctly
>>>>>>> master
```
So what it needs to be in the end is this:
```
    key1: This is the correct translation
```
I hope most of it is clear to you now ✌️

Reply:
Ok thanks you! :D

Reply:
I think it's good now...